docs/components/renderless/kstate.md

```diff
@@ -0,0 +1,78 @@
+# KState
+
+Holds state for presentation components.
+
+e.g.
+
+- counter
+- selected value
+
+<KState :initial-state="{ count: 0 }">
+  <div slot-scope="{ state, update }">
```

Comment:
should use v-slot instead e.g. `<KState v-slot="{state}"`

Reply:
`v-slot can only be used on components or <template>.` 🤷

Reply:
KState is a component